### ISSUE TITLE
Incorrectly formatted bold text inside a quote

### ISSUE THREAD

**Issue by zamuz** [silverbulletmd/silverbullet#792]
When bold text is at the beginning of a quote, it is formatted incorrectly:

https://github.com/silverbulletmd/silverbullet/assets/731751/db147ece-4805-43fb-80a5-2568b54ff19f

Might be related to the custom admonitions syntax that was added recently (https://github.com/silverbulletmd/silverbullet/pull/786).

**Comment by zamuz** [comment]
This issue appears to be back on version 0.8.4.

**Comment by janssen-io** [comment]
Although blockquotes has many issues still, I can't reproduce this bold-styling bug in 0.10.4
Do you still experience this issue, @zamuz ?

**Comment by zamuz** [comment]
@janssen-io Yes, as you described in your edit the issue persists on 0.10.4. The bold text is visible, but the quote formatting is still incorrect.

**Comment by janssen-io** [comment]
It seems to be related to recognizing admonitions. It then pushes the text to the right to make space for the admonition's icon: 

![Image](https://github.com/user-attachments/assets/948ac19e-3321-41bc-a83e-2982e7f8bac0)

A quick fix would be to set the width to 0 by default and set a custom width in the admonition's [styles](https://silverbullet.md/Markdown/Admonitions). The disadvantage is that you need to duplicate that property for every defined admonition, because you can't know beforehand which ones are defined.

I'll whip up a PR to see if it's a feasible fix. :)

**Comment by janssen-io** [comment]
The style is still not consistent, but I think it's a good middle ground between code duplication and consistency for now. The way quotes are handled probably needs a revamp. :D 

![Proposed styling for quotes](https://private-user-images.githubusercontent.com/33344942/423251816-b538d619-a15a-4359-9029-625761a4dfe9.png?jwt=eyJhbGciOiJIUzI1NiIsInR5cCI6IkpXVCJ9.eyJpc3MiOiJnaXRodWIuY29tIiwiYXVkIjoicmF3LmdpdGh1YnVzZXJjb250ZW50LmNvbSIsImtleSI6ImtleTUiLCJleHAiOjE3NDIxNjA5ODYsIm5iZiI6MTc0MjE2MDY4NiwicGF0aCI6Ii8zMzM0NDk0Mi80MjMyNTE4MTYtYjUzOGQ2MTktYTE1YS00MzU5LTkwMjktNjI1NzYxYTRkZmU5LnBuZz9YLUFtei1BbGdvcml0aG09QVdTNC1ITUFDLVNIQTI1NiZYLUFtei1DcmVkZW50aWFsPUFLSUFWQ09EWUxTQTUzUFFLNFpBJTJGMjAyNTAzMTYlMkZ1cy1lYXN0LTElMkZzMyUyRmF3czRfcmVxdWVzdCZYLUFtei1EYXRlPTIwMjUwMzE2VDIxMzEyNlomWC1BbXotRXhwaXJlcz0zMDAmWC1BbXotU2lnbmF0dXJlPTMxMzRmOGJmMDRiOTA4ZTU3OGIyNjc3NTU0NzdjZjNjMjYzYmJiOWIzOTAyZWNhODdmMmI0OWViNWM3MzIyZjkmWC1BbXotU2lnbmVkSGVhZGVycz1ob3N0In0.10PCt3-EOavACJgQopnFThwurVwCpKj2GMsyjTtPjOw)
